net.danielgm.CloudCompare.yaml

```diff
@@ -0,0 +1,290 @@
+app-id: net.danielgm.CloudCompare
+runtime: org.kde.Platform
+runtime-version: "5.12"
+sdk: org.kde.Sdk
+command: CloudCompare
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --share=network
+  - --filesystem=home
+  - --filesystem=host
+
+cleanup:
+    - /bin/applygeo
+    - /bin/cgal_*
+    - /bin/CreateDOMDocument
+    - /bin/DOM*
+    - /bin/EnumVal
+    - /bin/flann*
+    - /bin/gdal*
+    - /bin/geotifcp
+    - /bin/gnmanalyse
+    - /bin/gnmmanage
+    - /bin/listgeo
+    - /bin/MemParse
+    - /bin/nearblack
+    - /bin/ogr*
+    - /bin/pdal*
+    - /bin/PParse
+    - /bin/PSVIWriter
+    - /bin/Redirect
+    - /bin/SAX*
+    - /bin/SCMPrint
+    - /bin/SEnumVal
+    - /bin/StdInParse
+    - /bin/testepsg
+    - /bin/XInclude
+
+    - /lib/cmake
+    - /lib/debug
+    - /lib/gdalplugins
+    - /lib/pkgconfig
+    - "*.a"
+    - "*.la"
+
+    - /share/doc
+    - /share/eigen3
+    - /share/flann
+    - /share/gdal
+    - /share/info
+    - /share/man
+    - /share/pcl-1.9
+    - /share/pkgconfig
+    - /share/proj
+
+    - /include
+
+modules:
+  - name: MPFR
+    sources:
+      - type: archive
+        url: https://www.mpfr.org/mpfr-current/mpfr-4.0.2.tar.xz
+        sha256: 1d3be708604eae0e42d578ba93b390c2a145f17743a744d8f3f8c2ad5855a38a
+    buildsystem: autotools
+    builddir: true
+
+  - name: boost
+    sources:
+      - type: archive
+        url: https://dl.bintray.com/boostorg/release/1.69.0/source/boost_1_69_0.tar.gz
+        sha256: 9a2c2819310839ea373f42d69e733c339b4e9a19deab6bfec448281554aa4dbb
+    buildsystem: simple
+    build-commands:
+    - ./bootstrap.sh --prefix=/app --with-libraries=thread,system,date_time,filesystem,iostreams,atomic,chrono
+    - ./b2 install -j8
+
+  - name: CGAL
+    sources:
+      - type: archive
+        url: https://github.com/CGAL/cgal/releases/download/releases%2FCGAL-4.14/CGAL-4.14.tar.xz
+        sha256: 59464b1eaee892f2223ba570a7642892c999e29524ab102a6efd7c29c94a29f7
+    buildsystem: cmake
+    builddir: true
```

Comment:
```suggestion

```

Reply:
Why should I remove `builddir`? CMake complains when building directly in the source tree.

Reply:
My bad. It's set to true by default for meson.